sdk/storage/azure-storage-blob/azure/storage/blob/_blob_service_client.py

```diff
@@ -405,11 +400,10 @@ def set_service_properties(
 
     @distributed_trace
     def list_containers(
-            self, name_starts_with=None,  # type: Optional[str]
-            include_metadata=False,  # type: Optional[bool]
-            **kwargs
-        ):
-        # type: (...) -> ItemPaged[ContainerProperties]
+        self, name_starts_with: Optional[str] = None,
+        include_metadata: Optional[bool] = False,
```

Comment:
Hmm is this actually `Optional` just because it includes a default value? I think this type is still just `bool` since I think `Optional` just means it can be `None`? I feel like we probably do this all over the place but now I'm questioning it :D